docs/usage.rst

```diff
@@ -0,0 +1,101 @@
+=====
+Usage
+=====
+
+Used at the command line to auto-generate an INSERT statement.  The generated
+statement will generally require some hand-editing for your own purposes, but
+the auto-generated column list and comments indicating destinations should save
+you tedium and errors.
+
+Specifying database
+-------------------
+
+- Accepts `SQLAlchemy database URLs <http://docs.sqlalchemy.org/en/latest/core/engines.html>`_ with `--db` option.
+Defaults to environment variable `$DATABASE_URL`
+
+Generally used from the command line.
+
+Sample data
+-----------
+
+To try the examples in your own PostgreSQL instance, run the data setup SQL commands from
+`here <sample_tables.sql>`_.
```

Comment:
This link name isn't very accessible--full details are in the [18F content guide](https://content-guide.18f.gov/urls-and-filenames/#link-text). Maybe it'd be better to write something like:

```
To try the examples in your own PostgreSQL instance, run the `sample data setup SQL commands <sample_tables.sql>`_.
```

Reply:
Great catch, thank you!